rust/tw_coin_registry/src/tw_derivation.rs

```diff
@@ -16,7 +16,9 @@ pub enum TWDerivation {
     BitcoinTestnet = 4,
     LitecoinLegacy = 5,
     SolanaSolana = 6,
-    /// Default derivation.
+    StratisSegwit = 7,
```

Comment:
Looks like this one is not covered by tests, also what is the difference BitcoinSegwit vs StratisSegwit? I see no new entries in registry.json for this derivation

Reply:
This is the legacy behaviour of our codegen tool. Let's say, there is `segwit` derivation for Bitcoin and Stratis chains, then `codegen` will generate `BitcoinSegwit` and `StratisSegwit` accordingly.
I tried to change this behaviour to have just `Segwit` for both bitcoin and other chains, but had to revert back here https://github.com/trustwallet/wallet-core/pull/3965. That's because the derivation enum variant is stored in `StoredKey`, and changing the enum variants will break backward compatibility.

Regarding `StratisSegwit` tests - Stratis chain has been added when we merged Binance's WC into origin repo. We don't use it. Even though, `StratisSegwit` actually equals to `BitcoinSegwit`, so no explicit tests required

Reply:
gotcha, thanks @satoshiotomakan!